client/src/app/components/forms/inputs/types/docker-input.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, OnInit} from "@angular/core";
+
+@Component({
+    selector: 'docker-input',
+    template: `
+         <input name="dockerPull"
+                type="text"
+                class="form-control"
+                id="dockerImage"
+                [(ngModel)]="dockerPull">
+    `,
+})
+export class DockerInput implements OnInit {
```

Comment:
Docker is something I'd really only see being used in one place: in the docker section of a command line tool. So long as we have a form, its inputs don't have to be isolated as well.